developer-docs-site/docusaurus.config.js

```diff
@@ -127,6 +127,40 @@ const config = {
         additionalLanguages: ["rust"],
       },
     }),
+  plugins: [
+    [
+      '@docusaurus/plugin-client-redirects',
+      {
+        // fromExtensions: ['html', 'htm'], // /myPage.html -> /myPage
+        // toExtensions: ['exe', 'zip'], // /myAsset -> /myAsset.zip (if latter exists)
+        redirects: [
+          // /docs/oldDoc -> /docs/newDoc
+          {
+            to: '/tutorials/full-node/run-a-fullnode',
+            from: '/tutorials/run-a-fullnode',
+          },
+          // Redirect from multiple old paths to the new path
+          /*
+          {
+            to: '/docs/newDoc2',
+            from: ['/docs/oldDocFrom2019', '/docs/legacyDocFrom2016'],
+          },
+          */
+        ],
+        /*
+        createRedirects(existingPath) {
+          if (existingPath.includes('/community')) {
+            // Redirect from /docs/team/X to /community/X and /docs/support/X to /community/X
+            return [
+              existingPath.replace('/community', '/docs/team'),
+              existingPath.replace('/community', '/docs/support'),
+            ];
+          }
+          return undefined; // Return a falsy value: no redirect created
+        }, */
```

Comment:
let's clean up all this commented out code.

Reply:
yes.